superset/config.py

```diff
@@ -264,6 +264,12 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 # Configuration for scheduling queries from SQL Lab.
 SCHEDULED_QUERIES: dict[str, Any] = {}
 
+# Rate limiting
```

Comment:
```suggestion
# FAB rate limiting
```

Maybe there's merit in mentioning FAB so it's clear where these limits are being enforced.